static/app/views/performance/landing/widgets/transforms/transformMetricsToVitalSeries.tsx

```diff
@@ -0,0 +1,51 @@
+import moment from 'moment';
+
+import {getParams} from 'sentry/components/organizations/globalSelectionHeader/getParams';
+import {defined} from 'sentry/utils';
+import {MetricsRequestRenderProps} from 'sentry/utils/metrics/metricsRequest';
+import {DEFAULT_STATS_PERIOD} from 'sentry/views/performance/data';
+
+import {QueryDefinitionWithKey, WidgetDataConstraint, WidgetPropUnion} from '../types';
+
+export function transformMetricsToVitalSeries<T extends WidgetDataConstraint>(
+  widgetProps: WidgetPropUnion<T>,
+  results: MetricsRequestRenderProps,
+  _: QueryDefinitionWithKey<T>
+) {
+  const {start, end, utc, interval, statsPeriod} = getParams(widgetProps.location.query, {
+    defaultStatsPeriod: DEFAULT_STATS_PERIOD,
+  });
+
+  const metricsField = widgetProps.Queries.chart.fields[0];
+
+  const {errored, loading, reloading, response} = results;
+
+  const data = response?.groups.map(group => {
```

Comment:
should we default to` [] `here in case the response is `undefined`?

Reply:
no need, both are handled in `_VitalChart`